### DISCUSSION TITLE
MIMIC IV Echo Data

### DISCUSSION THREAD

**Discussion by JBarsotti** [MIT-LCP/mimic-code#1528] [MIMIC-General]
### Description

You all are doing wonderful work with MIMIC! My collaborators are cardiologists and would be interested in getting their hands on MIMIC-IV-ECG. I know it comes out in August but also know that it is available to the consortium members. I was just wondering whether there is a process to become a consortium member and whether it is worth it.

Thanks!

**Comment by alistairewj** [comment]
You can reach out to consortium@physionet.org for more detail and those responsible for the consortium will get back to you I'm sure.

**Reply by JBarsotti** [reply]
Thank you! I am working on a presentation using your MIMIC data right now. :)

**Comment by emiramaro** [comment]
Hi, my colleagues and I are currently looking at the MIMIC-IV echocardiogram dataset. We are trying to use the echocardiogram data and linking it to the MIMIC-IV Note. It is mentioned that the note will be made available on the MIMIC-IV Note for the studies that have note_id within the ‘study_list.csv’. However, upon examination on the MIMIC-IV Note ‘radiology.csv’ dataset, we did not find any linked report to the note_id within the echocardiogram dataset. We did find some echocardiogram descriptions within the ‘discharge.csv’ dataset in a form of discharge summaries.
 
Thus, is the cardiology report for the dataset has not been made available within the MIMIC-IV note? Or is the one within ‘discharge.csv’ would be the echo report?

Thank you!

**Comment by briangow** [comment]
@emiramaro , the cardiologist notes for a given Echo have not been released yet. We are working on getting those released, hopefully it'll be sooner than later.

**Reply by emiramaro** [reply]
Thank you for the response, Brian.

**Reply by WangMengxiao319** [reply]
@briangow, thanks for your elaborate work! I am also looking forward to the cardiologist notes relevant to Echo.

**Reply by ankushjindal5** [reply]
@briangow Hey Brian, wondering when the notes would be released, I believe the MIMIC echo data has been released for quite a long time, it would be very helpful if those scans are complemented by their reports. Truly, It would be a game changer for Medical AI community. Can you provide a timeline for the release ?

**Reply by briangow** [reply]
@ankushjindal5 , we are actively working on this but don't have an estimated release date at this point.

**Reply by adgilbert** [reply]
Hey @briangow just wanted to check in and see if you have an update on the release of the cardiology notes. Thanks for this amazing resource!

**Reply by briangow** [reply]
@adgilbert , unfortunately the notes aren't ready to be released. It will likely be some more time before we complete updates to our pipeline and process the notes. I do expect the structured measurements to be available sooner than the free-text notes.

**Reply by michelleespranita** [reply]
Hi @briangow! I was wondering when the structured measurements of the echos would be released. Could we expect them to be released within this year? Thanks for your hard work in making this resource available for the community!

**Reply by briangow** [reply]
@michelleespranita , we are preparing the release of the structured measurements now. They should be available by the end of the year. The free text notes will not be released at this time though.

**Reply by gangli6** [reply]
hi @briangow, I'd like to inquire about the release plan of the cardiovascular notes for a given echo. Will it be in the first half of 2026? Thank you for providing this valuable resource!

> @michelleespranita , we are preparing the release of the structured measurements now. They should be available by the end of the year. The free text notes will not be released at this time though.我们正在准备发布结构化测量数据，预计将于年底前发布。但自由文本说明暂不发布。

**Reply by samkleeman1** [reply]
Hi @briangow I am also interested to follow up reagrding the release of structured measurements for this cohort. Thank you, Sam Kleman

**Reply by michelleespranita** [reply]
Hi @briangow, I would like to follow up regarding the structured measurements. Are there any plans to release them soon? Thank you in advance!

**Reply by samkleeman1** [reply]
I am also keen to follow-up @briangow

**Comment by Rapuris** [comment]
Hello, I am working with the MIMIC-IV Echocardiogram dataset. However, I am unable to find out how exactly the different views within the dataset are represented. I could not find metadata within the dicom files suggesting what probe position/angle view number _XXXX corresponds to. Moreover, do the view numbers between different patients correspond to the same probe position/angle (i.e.,  do all views _0001 represent the same probe position across all patients? Any help would be greatly appreciated!

**Reply by Rapuris** [reply]
Following back up. Any advice would be greatly appreciated!

**Reply by briangow** [reply]
@Rapuris , sorry for the late reply. The Echo metadata doesn't contain information regarding the probe position/angle for a given view. The view numbers also do not necessarily correspond to a particular probe position/angle. 

We are hoping to release a derived dataset at some point which uses ML to classify the view information.

**Reply by Rapuris** [reply]
@briangow Thank you for the update! As of now, is there any way to distinguish between the videos/frames in the study (as to type of ultrasound imaging, view, etc.)? I assume not, but I want to confirm. I'm looking forward to the derived dataset as well. Any estimation of when that is coming out?

**Comment by briangow** [comment]
The structured measurements have been released in v1.0 of the project.

**Comment by briangow** [comment]
Closing this issue. Please open a new issue if you have mentioned something here that hasn't been addressed yet. This thread has become too broad to track effectively as one issue.
